lib/util/data-conversions.js

```diff
@@ -0,0 +1,37 @@
+exports.BufferEncodingStandards = {
+  UTF8 : 'utf8',
+  BYTES: 'bytes',
+}
+
+const UTF8 = exports.BufferEncodingStandards.UTF8
+const BYTES = exports.BufferEncodingStandards.BYTES
+
+const bytesBufferToListOfBytes = buffer => {
+  const lisOfBytes = []
+
+  for (let i = 0; i < buffer.length; i++) {
+    lisOfBytes.push(buffer[i])
+  }
+
+  return lisOfBytes
+}
+
+exports.convertDataFromTo = function(data, from, to) {
+  if (from === to) {
+    return data
+  }
+
+  if (typeof Buffer === 'undefined') {
```

Comment:
the coderunner runs always in the NodeJS env, it means there is always Buffer present, just require it